src/main/java/uk/gov/companieshouse/api/accounts/CompanyAccountsApplication.java

```diff
@@ -63,6 +63,6 @@ public void addInterceptors(final InterceptorRegistry registry) {
         registry.addInterceptor(companyAccountInterceptor)
                 .addPathPatterns("/transactions/{transactionId}/company-accounts/{companyAccountId}/**");
         registry.addInterceptor(smallFullInterceptor)
-                .addPathPatterns("/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/**");
+                .addPathPatterns("/transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/*");
```

Comment:
Will this catch longer URLs?
e.g. /transactions/{transactionId}/company-accounts/{companyAccountId}/small-full/notes/loans-to-directors/loans